lib/mongo/error/connection_unavailable.rb

```diff
@@ -1,27 +0,0 @@
-# frozen_string_literal: true
-# encoding: utf-8
-
-# Copyright (C) 2022 MongoDB Inc.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#   http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-module Mongo
-  class Error
-
-    # Exception raised when trying to check out a connection with a specific
-    # global id, and the connection for that global id no longer exists in the
-    # pool.
-    class ConnectionUnavailable < Error
```

Comment:
Is is a BC break? This class seems to be public.

Reply:
yeah good call